exporter/awsemfexporter/config.go

```diff
@@ -101,6 +101,8 @@ type MetricDescriptor struct {
 	// Overwrite set to true means the existing metric descriptor will be overwritten or a new metric descriptor will be created; false means
 	// the descriptor will only be configured if empty.
 	Overwrite bool `mapstructure:"overwrite"`
+	// Set the storage resolution for this metric in AWS Embedded Metric Format (EMF). Valid values are 1 (for 1 second resolution) and 60 (for 60 second resolution).
+	StorageResolution int `mapstructure:"storage_resolution"`
```

Comment:
The `MetricDescriptor` configuration is extremely limited and can be accomplished through generally-applicable processors.  I think it should be deprecated and slated for removal at some point in the future so we should not expand its surface area any further.

Reply:
The latest update instead looks for a metric attribute named `aws.emf.storage_resolution` which can be applied either in the application's code or using the Attribute Processor in the collector itself.